src/s3.py

```diff
@@ -0,0 +1,40 @@
+"""S3 connection utility classes and methods."""
```

Comment:
OUT OF SCOPE: for future reference please have a look at the new implementation of this class in the case of self-signed certificate

Reply:
Thanks.